Source/AliveLibAO/SaveGame.cpp

```diff
@@ -9,5 +15,232 @@ void SaveGame::sub_459970(BYTE*, int)
+    if (gElum_507680 != 0)
+    {
+        pSaveData->field_25C = sControlledCharacter_50767C == gElum_507680;
+        pSaveData->field_25E_elum_field_144 = gElum_507680->field_144;
+        pSaveData->field_28C_elum_continue_rect = gElum_507680->field_138_continue_rect;
+        pSaveData->field_294_continue_zone_number = gElum_507680->field_140_continue_zone_number;
+        pSaveData->field_296_elum_zone_number = gElum_507680->field_142_zone_number;
+        pSaveData->field_298_elum_continue_path = gElum_507680->field_148_continue_path;
+        pSaveData->field_29A_continue_level = gElum_507680->field_14A_continue_level;
+        pSaveData->field_29C_elum_sprite_scale = gElum_507680->field_150_continue_sprite_scale;
+        pSaveData->field_260_elum_lvl_number = gElum_507680->field_B2_lvl_number;
+        pSaveData->field_262_elum_path_number = gElum_507680->field_B0_path_number;
+        pSaveData->field_268_elum_xpos = FP_GetExponent(gElum_507680->field_A8_xpos);
+        pSaveData->field_26C_elum_ypos = FP_GetExponent(gElum_507680->field_AC_ypos);
+        if (gElum_507680->field_F4_pLine)
+        {
+            pSaveData->field_270 = gElum_507680->field_F4_pLine->field_8_type;
+        }
+        else
+        {
+            pSaveData->field_270 = -1;
+        }
+        pSaveData->field_274 = gElum_507680->field_FC_current_motion;
+        pSaveData->field_272_elum_flipX = gElum_507680->field_10_anim.field_4_flags.Get(AnimFlags::eBit5_FlipX);
+        pSaveData->field_278_brain_idx = gElum_507680->field_128_brain_idx;
+        pSaveData->field_276_elum_122 = gElum_507680->field_122;
+        pSaveData->field_27C_honey_xpos = gElum_507680->field_12C_honey_xpos;
+        pSaveData->field_27A_elum_brain_state = gElum_507680->field_12A_brain_state;
+        pSaveData->field_284_elum_field_130 = gElum_507680->field_130;
+        pSaveData->field_280_honey_ypos = gElum_507680->field_146;
+        pSaveData->field_288_elumflag1 = gElum_507680->field_170_flags.Get(Elum::Flags_170::eAttackedByBees_Bit1);
+        pSaveData->field_289_elumflag2 = gElum_507680->field_170_flags.Get(Elum::Flags_170::eBit2);
+        pSaveData->field_28A_elumflag3 = gElum_507680->field_170_flags.Get(Elum::Flags_170::eBit3);
+        pSaveData->field_28B_elumflag4 = gElum_507680->field_170_flags.Get(Elum::Flags_170::eFoundHoney_Bit4);
+    }
+    if (sGasTimer_507700)
+    {
+        pSaveData->field_2A8 = (gnFrameCount_507670 - sGasTimer_507700 <= 1) ? 1 : gnFrameCount_507670 - sGasTimer_507700;
+    }
+    else
+    {
+        pSaveData->field_2A8 = 0;
+    }
+    pSaveData->field_2AC = bUseAltSaveHeader_5076B4;
+    pSaveData->field_2AE = sCurrentControllerIndex_5076B8;
+    gMap_507BA8.SaveBlyData_446900(pSaveData->field_2B0_pSaveBuffer);
+    int* table = &pSaveData->field_204_zone_number;
+    int counter = 0;
+    for (int i = 1919; i > 0; i--)
+    {
+        table += 4;
+        counter += *table;
```

Comment:
its summing the 1919 fields after 200 and set it in 200, so seems like some pesudo crc/hash/anti save tampering thing